### ISSUE TITLE
"Flag Unit" on loyals doesn't work

### ISSUE THREAD

**Issue by Toranks** [knyghtmare/Unitmarker#2]
1.17.10
Click the option doesn't do anything if the selected unit is loyal.

**Comment by Toranks** [comment]
Problem is that this doesn't work, because overlays allways returns empty. Need a different filter. I'm searching for new method, because this can broke one of my addons too.
```
                            [filter_wml]
                                overlays="misc/loyal-icon.png"
                            [/filter_wml]
```
                            
![imagen](https://user-images.githubusercontent.com/40789364/209219144-df9710a7-71d2-44bc-af7c-c9516bd53612.png)

**Comment by Toranks** [comment]
I've changed the filter method and now works with leaders, loyals and normals. Heroes didn't work, neither before nor now, because filtering overlays is now done in a different way and I still don't know how.
https://github.com/knyghtmare/Unitmarker/pull/3
Edit: Filtered by ellipse instead of overlays. Works with Delfador on tutorial, for now, but some unique heroes may have different ellipses.

**Comment by knyghtmare** [comment]
Closed by #3
